substrate/frame/identity/src/lib.rs

```diff
@@ -537,6 +541,12 @@ pub mod pallet {
 				SubsOf::<T>::insert(&sender, (new_deposit, ids));
 			}
 
+			Self::deposit_event(Event::SubIdentitiesSet {
+				main: sender.clone(),
```

Comment:
```suggestion
				main: sender,
```

Shouldn't be needed right? Nothing below needs it?